### PR TITLE
Update url-resolver.md

### PR DESCRIPTION
**Pull request by RakeshJesadiya**
Add Example of the Category page and CMS page URL Resolver Queries.

## Purpose of this pull request

This pull request (PR) contains the additional example of the Category page and CMS Page queries. For the Category Page you have to pass `url` value will be full url of the category path.

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.3/graphql/queries/url-resolver.html
https://devdocs.magento.com/guides/v2.4/graphql/queries/url-resolver.html

### CODE REVIEW

File: src/guides/v2.3/graphql/queries/url-resolver.md [hunk 1]
```diff
@@ -47,13 +47,77 @@ The following query returns information about the URL containing `joust-duffle-b
...
+      "id": 4,
+      "relative_url": "gear/bags.html",
+      "redirectCode": 0,
+      "type": "CATEGORY"
+    }
+  }
+}
+```
+
+### URL Resolver for the Category type
```
Comment (BarnyShergold) [inline src/guides/v2.3/graphql/queries/url-resolver.md:82]: You have a duplicated title here - should it be, "URL resolver for CMS page" ?

File: src/guides/v2.3/graphql/queries/url-resolver.md [hunk 2]
```diff
@@ -47,13 +47,77 @@ The following query returns information about the URL containing `joust-duffle-b
...
+```
+
 ## Input attributes
 
 The `urlResolver` query contains the following attribute.
 
 Attribute | Type | Description
 --- | --- | ---
-`url` | String | The requested URL. To query for product and category pages, the `url` value must contain the URL key and suffix. For CMS page queries, the `url` value must contain the URL key only.
+`url` | String | The requested URL. To query for product and category pages, the `url` value must contain the URL key and suffix. For Category page queries. the `url` value must contain the full path of the URL key. For CMS page queries, the `url` value must contain the URL key only.
```
Comment (BarnyShergold) [inline src/guides/v2.3/graphql/queries/url-resolver.md:120]:
```suggestion
`url` | String | The requested URL. To query for product and category pages, the `url` value must contain the URL key and suffix. For Category page queries, the `url` value must contain the full path of the URL key. For CMS page queries, the `url` value must contain the URL key only.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Nice updates, just a few suggestions

File: src/guides/v2.3/graphql/queries/url-resolver.md (hunk 1 above)
Comment (RakeshJesadiya) [reply src/guides/v2.3/graphql/queries/url-resolver.md:82]: I will do changes.

**Comment by RakeshJesadiya** [comment]
@BarnyShergold I have made update based on your suggestion. Thank you.

**Review by BarnyShergold — APPROVED** [review]
Excellent!

**Comment by erikmarr** [comment]
@keharper can you take a look at this?

**Review by keharper — APPROVED** [review]

**Pull request merged**